packages/components/src/Button/Button/Button.module.scss

```diff
@@ -33,47 +33,50 @@
     text-decoration: none;
   }
 
-  &:not(:disabled) {
+  &:not(.disabled, .working) {
     &:active {
       transform: translateY(1px);
     }
+  }
 
-    &:focus {
-      outline: none;
-    }
+  &:focus {
+    outline: none;
+  }
 
-    &:focus-visible {
-      &::after {
-        $focus-ring-offset: calc(
-          (#{$border-focus-ring-border-width} * 2) + 1px
-        );
-
-        content: "";
-        position: absolute;
-        background: transparent;
-        border-color: $color-blue-500;
-        border-radius: $border-focus-ring-border-radius;
-        border-width: $border-focus-ring-border-width;
-        border-style: $border-focus-ring-border-style;
-        inset: calc(-1 * #{$focus-ring-offset});
-      }
+  &:focus-visible {
+    &::after {
+      $focus-ring-offset: calc((#{$border-focus-ring-border-width} * 2) + 1px);
+
+      content: "";
+      position: absolute;
+      background: transparent;
+      border-color: $color-blue-500;
+      border-radius: $border-focus-ring-border-radius;
+      border-width: $border-focus-ring-border-width;
+      border-style: $border-focus-ring-border-style;
+      inset: calc(-1 * #{$focus-ring-offset});
     }
   }
 
   &.reversed {
-    &:not(:disabled) {
-      &:focus-visible {
-        &::after {
-          border-color: $color-blue-300;
-        }
+    &:focus-visible {
+      &::after {
+        border-color: $color-blue-300;
       }
     }
   }
 
-  &:disabled,
+  &.disabled,
   &.working {
-    pointer-events: none;
+    cursor: default;
     opacity: 30%;
+
+    &:focus-visible {
+      &::after {
+        border-style: $border-dashed-border-style;
+        border-color: $color-gray-400;
```

Comment:
This looks way more washed out than the dashed borders we did for the disabled buttons on RTE (they used purple 800).

If you have local running: http://localhost:6006/?path=/story/components-richtexteditor-subcomponents-toggleiconbutton--sticker-sheet-default

Just checking this is on purpose?

Reply:
good spot, @HeartSquared if I recall correctly you've made this change. I guess it's based on a figma, can you please share the link to it?

Reply:
I don't recall unfortunately. I may have copied it from here with the plan to check later? Feel free to change it.
https://github.com/cultureamp/kaizen-design-system/blob/main/packages/components/src/Filter/FilterMultiSelect/subcomponents/SelectionControlButton/SelectionControlButton.module.scss#L68